src/components/EmployeesList/EmployeesList.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import { v4 as uuidv4 } from 'uuid';
+
+export default function EmployeesList() {
+  function createData(
+    name,
+    department,
+    position,
+    email,
+    mobPhone,
+    internalPhone
+  ) {
+    return { name, department, position, email, mobPhone, internalPhone };
+  }
+
+  const rows = Array(10)
+    .fill()
+    .map(() =>
+      createData(
+        'Абрамова Александра Андреевна',
+        'Отдел материально-технического снабжения',
+        'Торговый представитель',
+        'aabramova@logoipsum.ru ',
+        '+79093334455',
+        '11-11'
+      )
+    );
```

Comment:
```suggestion
    .map(() => ({
        name: 'Абрамова Александра Андреевна',
        department: 'Отдел материально-технического снабжения',
        position: 'Торговый представитель',
        email: 'aabramova@logoipsum.ru ',
        mobilePhone: '+79093334455',
        internalPhone: '11-11'
      })
    );
```
а ту функцию удалить, будет меньше кода и читателю надо будет меньше бегать глазами. Вообще, если проще заглянуть внутрь кода функции, чем написать документацию к ней, то это плохая функция, и ей будет тяжело пользоваться людям